Src/IronPython.Modules/IronPython.Modules.csproj

```diff
@@ -14,7 +14,7 @@
     <PackageReference Include="Mono.Posix" Version="5.4.0.201" Condition=" '$(Mono)' != 'true' " />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Mono.Posix.NETStandard" Version="1.0.0" Condition=" '$(TargetFramework)' == 'netcoreapp2.0' OR '$(TargetFramework)' == 'netcoreapp2.1' " />
```

Comment:
Can we just change this to something like

```
Condition=" $(TargetFramework.StartsWith('netcoreapp')) "
```

So we don't have to keep modifying this with new versions?

Reply:
Sure, we can always change them back to specific versions if we find it no longer works in the future.